cmd/process-agent/api/server.go

```diff
@@ -0,0 +1,55 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	settingshttp "github.com/DataDog/datadog-agent/pkg/config/settings/http"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/gorilla/mux"
```

Comment:
Move this import block to be above project imports (starting with ddconfig)

```suggestion
	"github.com/gorilla/mux"

	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
	settingshttp "github.com/DataDog/datadog-agent/pkg/config/settings/http"
	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Reply:
Oh gotcha I thought it was supposed to be below